src/Services/DatabaseTools/ORMSqliteDatabaseTool.php

```diff
@@ -166,6 +166,6 @@ protected function enableForeignKeyChecksIfApplicable(): void
 
     private function isSqlite(): bool
     {
-        return $this->connection->getDatabasePlatform() instanceof SqlitePlatform;
+        return $this->connection->getDatabasePlatform() instanceof SQLitePlatform;
```

Comment:
```suggestion
        return $this->connection->getDatabasePlatform() instanceof SqlitePlatform || $this->connection->getDatabasePlatform() instanceof SQLitePlatform;
```

Is this needed to keep compatibility?